### DISCUSSION TITLE
Task 4 Markdown tutorial

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#19] [Week 1 - Discussions]
After it created the index.md for me, I'm not able to edit it. It's like the editing feature is locked. Not sure where to go from here. I'm working through this tutorial: communicate-using-markdown under Task 4 week 1.

**Comment by metasean** [comment]
GitHub has been making a lot of tweaks to their interface recently, but a safe bet is always a "pencil" like icon.

<img width="1211" alt="Screen Shot 2023-05-13 at 2 30 01 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/7552754/e49523c6-9cdc-4768-80bf-c1063b35e3ee">

**Comment by Skye404** [comment]
<img width="891" alt="image" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/27c03895-d58d-485f-a101-8345532dc17c">
I'm not seeing a pencil option within the index.md file. It's not allowing me to edit the index.md file because the editing feature is blocked somehow. Can I get to the index.md file from another page?

**Reply by bst3ck** [reply]
So I think you are viewing this in the "pull request" tab in the "files changed section". You will be able to see the edit pencil if you click on the "code" tab.

**Reply by Skye404** [reply]
I was able to figure it out. I appreciate the help!
